### PR TITLE
Set up semaphore

### PR DESCRIPTION
**Pull request by kayoub5**
@chmorgan you just need a free semaphoreci.com account, the configuration is already tested.

### CODE REVIEW

File: SharpPcap/SharpPcap.csproj [hunk 1]
```diff
@@ -14,6 +14,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="PacketDotNet" Version="1.3.0" />
+    <PackageReference Include="PacketDotNet" Version="1.2.0" />
```
Comment (kayoub5) [inline SharpPcap/SharpPcap.csproj:17]:
```suggestion
```

**Comment by chmorgan** [comment]
@kayoub5 it might make sense to squash the commits for the config file changes together.

**Comment by chmorgan** [comment]
@kayoub5 alright, set up a hobby organization (the free open source one only builds linux) but not sure how to get the online stuff to be ok with the config file. It seems like its prompting me to setup a workflow.

**Comment by kayoub5** [comment]
> @kayoub5 it might make sense to squash the commits for the config file changes together.

I will do that once you setup the account

**Comment by kayoub5** [comment]
@chmorgan let me know once you create an account

**Comment by chmorgan** [comment]
@kayoub5 account created a bit ago. Added you to it as well

**Comment by kayoub5** [comment]
> 
> 
> @kayoub5 account created a bit ago. Added you to it as well

did you configure coverage token in it?

**Comment by kayoub5** [comment]
@chmorgan semaphore is working now, the only issue is that it's not uploading coverage due to lack of token configuration

**Pull request merged**